### PR TITLE
docs: Document the warm up query options

### PR DESCRIPTION
**Pull request by ptbrowne**
Add an example taken from banks on how to automatically fire warm up queries.

### CODE REVIEW

File: src/tutorials/data/pouchdb.md [hunk 1]
```diff
@@ -17,9 +17,54 @@ You will realize that Pouchdb operates in 2 phases: first, it loads candidate do
...
+import CozyPouchLink from 'cozy-pouch-link'
+
+const TRANSACTION_DOCTYPE = 'io.cozy.bank.operations'
+
+const makeWarmupQueryOptions = (doctype, indexedFields) => {
+  return {
+    definition: () => {
+      const qdef = Q(doctype)
+        .where(
+          fromPairs(indexedFields.map(fieldName => [fieldName, { $gt: null }]))
```
Comment (paultranvan) [inline src/tutorials/data/pouchdb.md:34]: The `fromPairs` import is missing ;)

File: src/tutorials/data/pouchdb.md [hunk 2]
```diff
@@ -17,9 +17,54 @@ You will realize that Pouchdb operates in 2 phases: first, it loads candidate do
...
+
+const client = new CozyClient({
+  links: [pouchLink, new CozyStackClient()]
+})
+```
+
+
+ℹ️ For advanced queries, you generally want to force it with the `use_index` option. 
+ℹ️ If there is a change in the underlying documents, the index will be partially recalculated on the next query.
+ℹ️ If the query and the index you force are not compatible, Pouch will emit an error and not run the query at all.
```
Comment (paultranvan) [inline src/tutorials/data/pouchdb.md:67]: Compared to before, we lost the `By default, Pouch will try to find the best index to use on your query.` info, which I find important to understand the first :information_source: . Furthermore, the third :information_source: is linked to the first, so I would rephrase like this:

ℹ️ By default, Pouch will try to find the best index to use on your query. But for advanced queries, you generally want to force it with the `use_index` option to be sure it does not mess up.
ℹ️ If the query and the index you force are not compatible, Pouch will emit an error and not run the query at all.
ℹ️ If there is a change in the underlying documents, the index will be partially recalculated on the next query.

**Review by paultranvan — APPROVED** [review]
Small remarks, but it's good, thanks :tada:

File: src/tutorials/data/pouchdb.md (hunk 1 above)
Comment (ptbrowne) [reply src/tutorials/data/pouchdb.md:35]: I added the import, thanks.
